From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Your Name <your.email@example.com>
Date: Thu, 22 Jan 2026 10:00:00 +0800
Subject: [PATCH] meson8b: pinctrl: add sd_a_pins node between sd_b_pins and sdxc_c_pins

Add missing sd_a_pins definition to fix compile error when referencing
sd_a_pins in SDIO subnodes. The node is placed between sd_b_pins and
sdxc_c_pins to follow the existing SD/SDXC pin group ordering in meson8b.dtsi.

Signed-off-by: Your Name <your.email@example.com>
---
 arch/arm/boot/dts/meson8b.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/meson8b.dtsi b/arch/arm/boot/dts/meson8b.dtsi
index 9f87a5678b12..a1b2c3d4e5f6 100644
--- a/arch/arm/boot/dts/meson8b.dtsi
+++ b/arch/arm/boot/dts/meson8b.dtsi
@@ -487,6 +487,15 @@
 				bias-disable;
 			};
 		};
+		
+		sd_a_pins: sd-a {
+			mux {
+				groups = "sd_d0_a", "sd_d1_a", "sd_d2_a",
+					"sd_d3_a", "sd_clk_a", "sd_cmd_a";
+				function = "sd_a";
+				bias-disable;
+			};
+		};
 
 		sdxc_c_pins: sdxc-c {
 			mux {

-- 
2.43.0
